CMakeLists.txt

```diff
@@ -0,0 +1,328 @@
+########################################################################################
+#
+# This is the install script for ILLIXR
+# This will compile the main ILLIXR binary along with any requested plugins.
+# The currently supported plugins are:
+#   audio_pipeline, debugview, gldemo, ground_truth_slam, gtsam_integrator, kimera_vio,
+#   monado, offline_cam, offline_imu, offload_data, openxr_app, orb_slam, pose_prediction,
+#   realsense, timewarp_gl
+#
+# In addition to the standard CMake options, the following arguments can be used:
+#
+# BUILD_GROUP [ALL|NATIVE|MONADO|CI|TESTS|NONE] : which subgroup of plugins are to be built
+#     ALL : all plugins, this is the default
+#     NATIVE : audio_pipeline, debugview, gldemo, gtsam_integrator, ground_truth_slam,
+#              kimera_vio, offline_imu, offline_cam, offload_data, pose_prediction, timewarp_gl
+#     MONADO : audio_pipeline, gtsam_integrator, kimera_vio, monado, offline_imu, offline_cam,
+#              openxr_app, pose_prediction, timewarp_gl
+#     CI : audio_pipeline, gldemo, ground_truth_slam, gtsam_integrator, kimera_vio,
+#          offline_imu, offline_cam, pose_prediction,  timewarp_gl
+#     NONE : no group is built, individual plugins can be selected
+#
+# To build specific plugins use the following format:
+#   USE_<PLUGIN>=ON
+# where <PLUGIN> is the name of the plugin in upper case.
+#
+########################################################################################
+
+cmake_minimum_required(VERSION 3.16)
+set(CMAKE_VERBOSE_MAKEFILE True)
+if (POLICY CMP0048)
+    cmake_policy(SET CMP0048 NEW)
+endif (POLICY CMP0048)
+
+if (POLICY CMP0135)
+    cmake_policy(SET CMP0135 NEW)
+endif()
+
+if (POLICY CMP0077)
+    cmake_policy(SET CMP0077 NEW)
+endif()
+
+set(BUILDING_ILLIXR Yes)
+
+set(PACKAGE_NAME "ILLIXR")
+set(PACKAGE_STRING "ILLIXR 1.0.0")
```

Comment:
We can set this to 3.2.0 since this would be our next major release

Reply:
Done